common/src/main/java/com/example/android/uamp/media/UampNotificationManager.kt

```diff
@@ -0,0 +1,116 @@
+package com.example.android.uamp.media
```

Comment:
Looks like the license header is missing?

Reply:
Good spot, thanks.